qiskit/transpiler/passes/optimization/inverse_cancellation.py

```diff
@@ -0,0 +1,63 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+A generic gate-inverse cancellation pass for a broad set of gate-inverse pairs.
+"""
+
+from qiskit.transpiler.basepasses import TransformationPass
+
+
+class Cancellation(TransformationPass):
+    """Cancel back-to-back `gates`in dag."""
+
+    def __init__(self, gate_cancel):
+        """Initialize gate_cancel"""
+        self.gate_cancel = gate_cancel
+        print(self.gate_cancel)
```

Comment:
We forgot to remove debugging code
Before code is merged:
  - [ ] No print statements
  - [ ] No commented out code
  - [ ] Double checking for typos
  - [ ] All docstrings (documentation for methods, functions, classes, modules) have been updated/created